src/components/ObjectList.vue

```diff
@@ -114,7 +121,8 @@ export default {
       items: [],
       page: 1,
       perPage: 20,
-      total: 0
+      total: 0,
+      sortBy: [{ key: "name", order: "asc" }]
```

Comment:
Name for entities makes sense but for observables, imo that's not the best key. I would have chosen created for observables.

Reply:
Correct - the Observables vue uses another component (ObservableSearch) with a custom table so this only affects entities and indicators :)

Observables already sorts things by created time desc